conf/server/server_full.conf

```diff
@@ -534,6 +534,65 @@ plugins {
     #     }
     # }
 
+    # UpstreamAuthority "vault": Uses a PKI Secret Engine from HashiCorp Vault
+    # to sign SPIRE server intermediate certificates.
+    # UpstreamAuthority "vault" {
+    #     plugin_data {
+    #         # vault_addr: The URL of the Vault server.
+    #         # Default: ${VAULT_ADDR}.
+    #         # vault_addr = ""
+
+    #         # pki_mount_point: Name of the mount point where PKI secret engine is mounted.
+    #         # Default: pki.
+    #         # pki_mount_point = ""
+
+    #         # ca_cert_path: Path to a CA certificate file used to verify
+    #         # the Vault server certificate. Only PEM format is supported.
+    #         # Default: ${VAULT_CACERT}.
+    #         # ca_cert_path = ""
+
+    #         # insecure_skip_verify: If true, vault client accepts any server certificates.
+    #         # Default: false.
+    #         # insecure_skip_verify = ""
```

Comment:
Thank you for your review. I fixed as below:

> This is a boolean, so I think that is better if set the example value as false.

https://github.com/spiffe/spire/pull/1732/commits/3936d41e75689acfbc3f2d02261d4ac6b1073dcd

> It would be great if we can fix that as part of this PR.

https://github.com/spiffe/spire/pull/1732/commits/cbc5286a7d61e9e8c60d55efc7d8fc9e83a04854, https://github.com/spiffe/spire/pull/1732/commits/5c7865111ef4482c13389d09f5ec164205be4fcd